packages/gestalt/src/TagData.js

```diff
@@ -44,11 +44,7 @@ export type TileChangeHandler = ({|
 |}) => void;
 
 export type OnRemoveHandler = ({|
-  event:
-    | SyntheticMouseEvent<HTMLDivElement>
-    | SyntheticKeyboardEvent<HTMLDivElement>
-    | SyntheticMouseEvent<HTMLAnchorElement>
-    | SyntheticKeyboardEvent<HTMLAnchorElement>,
+  event: SyntheticMouseEvent<HTMLButtonElement>,
```

Comment:
Why this change?

Reply:
what if it's click from the keyboard?

Reply:
Same handler/implementation as`Tag`. It's an HTML button `onClick` handler, it doesn't give the keyboard events back